editor/editor_interface.cpp

```diff
@@ -336,6 +337,20 @@ void EditorInterface::popup_property_selector(Object *p_object, const Callable &
 	property_selector->connect(SNAME("canceled"), canceled_callback, CONNECT_DEFERRED);
 }
 
+void EditorInterface::popup_quick_open(const Callable &p_callback, const TypedArray<StringName> &p_base_types) {
+	Vector<StringName> base_types;
+	if (p_base_types.is_empty()) {
+		base_types.append("Resource");
+	}
+	for (int i = 0; i < p_base_types.size(); i++) {
+		base_types.append(p_base_types[i]);
```

Comment:
```suggestion
	for (const StringName &type : p_base_types) {
		base_types.append(type);
```

Reply:
This is another thing that fail CI even though it works fine.

Reply:
Maybe you need to use Variant as type.

Reply:
I don't see much benefit. TypedArrays are iterated with regular loops throughout the code base, including other PRs I did.

Reply:
Likely because range iteration on Array wasn't supported until recently.